src/app/dwg/libdxfrw/intern/drw_textcodec.cpp

```diff
@@ -549,7 +549,7 @@ std::string DRW_Conv932Table::toUtf8( std::string *s )
 std::string DRW_ConvUTF16::fromUtf8( std::string *s )
 {
   DRW_UNUSED( s );
-  //RLZ: to be writen (only needed for write dwg 2007+)
+  //RLZ: to be written (only needed for write dwg 2007+)
```

Comment:
this typo is already in spelling.dat, but we leave as is external libraries (libdxfrw)